src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -202,11 +202,11 @@ public async Task<GetTxOutModel> GetTxOutAsync(string txid, uint vout, bool incl
             UnspentOutputs unspentOutputs = null;
             if (includeMemPool)
             {
-                unspentOutputs = this.pooledGetUnspentTransaction != null ? await this.pooledGetUnspentTransaction.GetUnspentTransactionAsync(trxid) : null;
+                unspentOutputs = this.pooledGetUnspentTransaction != null ? await this.pooledGetUnspentTransaction.GetUnspentTransactionAsync(trxid).ConfigureAwait(false) : null;
```

Comment:
Why remove the `ConfigureAwait` ?

Reply:
Forgot to reintroduce it after making GetUnspentTransactionAsync not async and then changing my mind and putting it back.

Reply:
It was added in this PR actually